lua/lspconfig/ui/lspinfo.lua

```diff
@@ -247,11 +273,13 @@ return function()
   vim.cmd 'let m=matchadd("error", "false")'
   for _, config in pairs(configs) do
     vim.fn.matchadd('Title', '\\%(Client\\|Config\\):.*\\zs' .. config.name .. '\\ze')
-    vim.fn.matchadd('Visual', 'list:.*\\zs' .. config.name .. '\\ze')
+    vim.fn.matchadd('Function', 'list:.*\\zs' .. config.name .. '\\ze')
     if config.filetypes then
       for _, ft in pairs(config.filetypes) do
         vim.fn.matchadd('Type', '\\%(filetypes\\|filetype\\):.*\\zs' .. ft .. '\\ze')
```

Comment:
If we're adding `LspInfoBorder` and `LspInfoTips`, we can turn these into custom groups as well.